File: 3.5.c
#include<stdio.h>
int main(){
    int l,j;
    printf ("Length of the first and second array \n");
    scanf ("%d %d",&l,&j);
    int t[l],h[j],s[l+j];
    printf("the first\n");
    for (int i=0;i<l;i++){
        scanf("%d",&t[i]);
    }
    printf("the secend \n");
    for (int i=0;i<j;i++){
        scanf("%d",&h[i]);
    }
    for(int i=0;i<j;i++){
        s[i]=h[i];
    }
     for(int i=0;i<l;i++){
        s[i+j]=t[i];
    }
    printf("Merged array:\n");
    for (int i=0;i<l+j;i++){
     printf("\n %d",s[i]);
    }
    return 0;
}

File: 3.5.py
t = []
h = []
l=int(input("length og the first array \n"))
j=int(input("length og the secind array \n"))
for i in range (l):
    t.append(int(input(f"enter element {i+1} for the first array :")))
for i in range (j):
    h.append(int(input(f"enter element {i+1} for the secind array :")))
h+=t
print("Merged array:")
print(h)